common/djangoapps/util/date_utils.py

```diff
@@ -52,7 +53,7 @@ def get_time_display(dtime, format_string=None, coerce_tz=None):
         try:
             to_tz = timezone(coerce_tz)
         except UnknownTimeZoneError:
-            to_tz = UTC
+            to_tz = utc
         dtime = to_tz.normalize(dtime.astimezone(to_tz))
```

Comment:
The ambiguity exception issue occurs when creating timezone objects from scratch, not when normalizing.